### PR TITLE
Jekyll role

### PR DESCRIPTION
**Pull request by manics**
Copied the tasks for installing Jekyll and deploying a site from `jekyll-playbooks/`

### CODE REVIEW

**Comment by joshmoore** [comment]
Any objections to updating the playbooks to use the new role? cc @sbesson

**Comment by sbesson** [comment]
I was about to ask the same thing. No veto from my side if we are converting these playbooks into proper roles.

**Comment by manics** [comment]
I'm doing that in a separate PR

**Comment by sbesson** [comment]
I assume leaving out the nginx deployment part of the playbook was on purpose. In that case, it's unclear to me that `jekyll-site` is the right role name as this stops at the Jekyll build step. Otherwise, is there room for adding a nginx dependency with a default template? And could the latter be overriden in playbooks?

**Comment by manics** [comment]
Yes, it's intentional because there's too many ways to use nginx in production, e.g. serving a single jekyll site, serving multiple, serving jekyll and proxying a backend server, so I think it should be separate. I'm happy to rename `jekyll-site` to something else if you've got a suggestion.

File: ansible/test.yml [hunk 1]
```diff
@@ -95,6 +95,21 @@
       that:
       - "sphinx_version.stdout == 'Sphinx (sphinx-build) 1.2.3'"
 
+- hosts: jekyll
+  roles:
+  - role: jekyll-site
+    jekyll_site_name: test
+    jekyll_site_git_repo: https://github.com/lunson/www.openmicroscopy.org
```
Comment (sbesson) [inline ansible/test.yml]: We might want to change this to use an organization repo. IDR?

Comment (manics) [inline ansible/test.yml]: The proper www.openmicroscopy.org repo would make more sense, but as far as I can tell there's not much there yet.

Comment (sbesson) [inline ansible/test.yml]: That would work as well, we'll have to remember to update this test file eventually.

**Comment by manics** [comment]
Updated

**Comment by sbesson** [comment]
All works as expected both using Vagrant and the updated playbook in #173. @joshmoore any additional comment?

**Comment by joshmoore** [comment]
Not from me. :+1: Thanks, guys.

**Comment by sbesson** [comment]
Thanks @manics for the consolidation.

**Pull request merged**